### ISSUE TITLE
implementation of the training Loss

### ISSUE THREAD

**Issue by albertchristianto** [tensorboy/pytorch_Realtime_Multi-Person_Pose_Estimation#94]
Dear @tensorboy,

hello, my name is Albert Christianto and right now i am using your code for my research. I want to ask you about the implementation of the loss function in this repo.

if "COCO_val" in self.data[idx]['dataset']:
            mask_miss = cv2.imread(
                self.mask_dir + 'mask2014/val2014_mask_miss_' + img_idx + 'png', 0)
        elif "COCO" in self.data[idx]['dataset']:
            mask_miss = cv2.imread(
                self.mask_dir + 'mask2014/train2014_mask_miss_' + img_idx + 'png', 0)

I am quite confused. why are you using mask miss version? because if I i interpreted the paper correctly, It should be using mask the labelled person (mask_all) so we didn't penalize true negative data. Thank you for your attention.

best regards,
Albert Christianto
